web/impact/impact/v1/helpers/criterion_helper.py

```diff
@@ -20,14 +24,62 @@
 class CriterionHelper(ModelHelper):
     model = Criterion
 
-    REQUIRED_KEYS = [
-        "name",
-        "type",
-        "judging_round_id",
-        ]
+    REQUIRED_KEYS = ["name",
+                     "type",
+                     "judging_round_id"]
     ALL_KEYS = REQUIRED_KEYS
     INPUT_KEYS = ALL_KEYS
 
+    specific_helpers = {}
+
+    @classmethod
+    def register_helper(cls, klass, type, name):
+        cls.specific_helpers[(type, name)] = klass
+
+    @classmethod
+    def find_helper(cls, type, name):
+        return cls.specific_helpers.get((type, name), cls)
+
+    def options(self, spec, apps):
+        return [spec.option]
+
+    def app_ids_for_feedback(self, feedbacks, option_name, **kwargs):
+        return self.feedbacks_for_option(feedbacks, option_name).values_list(
+            "application_id", flat=True)
+
+    def app_count(self, apps, option_name):
+        return apps.count()
+
+    def total_commitments(self, commitments, option_name):
```

Comment:
see comment elsewhere, I think this should be `total_capacity`

Reply:
Changed.